src/core/qgsvectorlayer.cpp

```diff
@@ -4474,6 +4475,7 @@ void QgsVectorLayer::onSymbolsCounted()
   if ( mFeatureCounter )
   {
     mSymbolFeatureCountMap = mFeatureCounter->symbolFeatureCountMap();
+    mSymbolIdMap = mFeatureCounter->symbolFeatureIdMap();
```

Comment:
Is this required/used?

Reply:
Absolutely this is a core component.

Reply:
@m-kuhn can I leave this in for the moment or do you also want it removed?

Reply:
Remove it